packages/fluent-editor/src/ui/icons.config.ts

```
@@ -315,9 +315,10 @@
 
 export const DIVIDER_ICON = `<svg viewBox="0 0 18 18"><line x1="0" y1="9" x2="18" y2="9" stroke="currentColor" stroke-width="2"/></svg>`
 
-export const AI_ICON = `<svg viewBox="0 0 24 24.0023" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24.0022736" fill="none" customFrame="#000000">
+export const AI_ICON = `<div class="ql-ai-icon-box">
+<svg viewBox="0 0 24 24.0023" class="ql-ai-trigger-icon ql-ai-icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24.0022736" fill="none" customFrame="#000000">
 	<defs>
-		<linearGradient id="paint_linear_1" x1="2.50000095" x2="12.0000048" y1="0" y2="23.9999962" gradientUnits="userSpaceOnUse">
+		<linearGradient class="ql-ai-icon-linear" id="paint_linear_1" x1="2.50000095" x2="12.0000048" y1="0" y2="23.9999962" gradientUnits="userSpaceOnUse">
```

ESLint

[error] Unexpected tab character.

(style/no-tabs)